### PR TITLE
Add option to allow display of the flag in the form widget

### PR DESCRIPTION
**Pull request by DesLynx**
Provides a solution for https://github.com/odolbeau/phone-number-bundle/issues/105

### CODE REVIEW

**Comment by Nek-** [comment]
Hello @DesLynx , thank you very much for this contribution! It looks interesting. However please share with us why, how, and also some screen of how it looks. (any details will help us decide if yes or no we are going to use it).

It also adds a new extension as the requirement... I think mbstring is probably installed everywhere by default, but we should consider this anyway.

**Comment by DesLynx** [comment]
Hi!
Why: it provides a better look and feel. With no extra CSS nor Javascript.
How: it uses emojis.

Note: I removed the requirement for the mbstring extension as `giggsey/libphonenumber-for-php` already requires the `symfony/polyfill-mbstring`.

Here is a screenshot (in Firefox) for this config:
```php
$form = $this->createFormBuilder($obj)
    ->add('phone', PhoneNumberType::class, [
        'label'                      => 'Phone number',
        'required'                   => true,
        'widget'                     => PhoneNumberType::WIDGET_COUNTRY_CHOICE,
        'country_choices'            => [],
        'preferred_country_choices'  => ['FR', 'CH', 'BE', 'LU'],
        'country_display_emoji_flag' => true,
    ])
    ->getForm();
```
image

**Review by odolbeau — APPROVED** [review]
Looks really great for me! :ok_hand:

**Comment by Nek-** [comment]
@DesLynx thanks for the details, however, we need to add the `symfony/polyfill-mbstring` as well in this case.

File: README.md [hunk 1]
```diff
@@ -175,6 +175,12 @@ The option `country_display_type` can be specified to change the country dropdow
 | `display_country_full` (default)  |  United Kingdom (+44)  |
 | `display_country_short`           |  GB +44                |
 
+And with the option `country_display_emoji_flag` set to `true` (default is `false`) you can add the emoji flag of the country before the label :
```
Comment (Nek-) [inline README.md:178]: Why default false ? :)

Comment (Nek-) [reply README.md:178]: wdyt @odolbeau

Comment (DesLynx) [reply README.md:178]: If set to true then people updating the bundle risk to have their design break by the added emoji. With false by default the update won't break anything and people keep full control over their design.

Comment (DesLynx) [reply README.md:178]: IMO changing the default value to true may be done with a future major release and document as BC break. And even better with a deprecation notice triggered  before that ;)

Comment (Nek-) [reply README.md:178]: WDYT about setting it to true in the next major version?

Comment (odolbeau) [reply README.md:178]: Sure, why not. :)

Comment (DesLynx) [reply README.md:178]: Changelog updated

**Review by Nek- — APPROVED** [review]

**Review by maxhelias — APPROVED** [review]

**Comment by DesLynx** [comment]
Hi! I noticed that you haven't created a new tag for the repo since merging the PR. As it adds the info about the new option `country_display_emoji_flag` in the README this can be confusing for the users...

**Pull request merged**